pubsub/mqtt_connection.py

```diff
@@ -0,0 +1,49 @@
+import logging
+
+import redis
+
+from pubsub import Connection
+from paho.mqtt import client as mqtt_client
+logger = logging.getLogger('root')
+
+class MqttConnection(Connection):
+
+    def __init__(self, host='localhost', port='6379', user=None, password=None):
+        super().__init__(host, port, user, password)
+        self.is_connection = False
+
+    # Create a dummy connection to check if REDIS is available and future connections can be made
+    def connect(self):
+        def on_connection(user, userdata, flags, rc):
+            if rc == 0:
+                print("MQTT Broker has been connected!")
+            else:
+                print("Failed to connect, return code %d\n", rc)
+
+        client = mqtt_client.Client(self.user)
+        client.on_connect = on_connection
+        #print(self.host)
+        client.connect(self.host, self.port)
```

Comment:
what other values does it need, we should at least add authentication info here.